co2-sensor/main.py

```diff
@@ -0,0 +1,33 @@
+import subprocess
+import re
+import time
+
+from prometheus_client import CollectorRegistry, Gauge, push_to_gateway
+
+PUSHGATEWAY_ADDRESS = "localhost:9091"
```

Comment:
環境変数にしておいても良いかもしれません。

```suggestion
import os
PUSHGATEWAY_ADDRESS = os.getenv("PUSHGATEWAY_ADDRESS", "localhost:9091")
```